recipes/ilamb/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "ILAMB" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15c2e0da0139d406df5883dad428ab8692aeebbb55368db0ed3b93466f9d19db
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python =2
```

Version constraints on `python` should only be used with `noarch: python` packages. The `skip: True` selector above will enforce the version requirement for this package.

```suggestion
    - python
```

(This package can't be a noarch as there is a `scripts` entry in the `setup.py`, if they can be changed to `entry_points` in a future release it would remove this limitation.)

---

Sounds good, I'll remove it -- I wasn't sure if it was needed in addition to the skip.